packages/eslint-plugin/tests/fixtures/tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "module": "commonjs",
     "strict": true,
     "esModuleInterop": true,
-    "lib": ["es2015", "es2017", "esnext"],
+    "lib": ["es2015", "es2017", "esnext", "DOM"],
```

Comment:
this fixes the `HTMLElement` tests

Reply:
This seems like it needs further clarification from #9648